client/components/upgrades/gsuite/gsuite-upsell-card/index.jsx

```diff
@@ -85,10 +86,12 @@ const GSuiteUpsellCard = ( {
 			<CompactCard>
 				<header className="gsuite-upsell-card__header">
 					<h2 className="gsuite-upsell-card__title">
-						{ translate( 'Add professional email from G Suite by Google Cloud to %(domain)s', {
+						{ translate( 'Add professional email from %(productFamily)s to %(domain)s', {
 							args: {
 								domain,
+								productFamily: getGoogleMailServiceFamily( productSlug ),
```

Comment:
The `productSlug` parameter will be correctly handled on https://github.com/Automattic/wp-calypso/pull/48761 has been merged.